### DISCUSSION TITLE
How to add a role under author in revealjs / pptx / pdf?

### DISCUSSION THREAD

**Discussion by k-soomro** [quarto-dev/quarto-cli#7110] [Q&A] [answered]
### Description

I would like to add a role under the author name in different formats including revealjs, pptx and pdf, like this:

Title
Author name
Position

Any suggestions on how I can do this?

**Comment by mcanouil** [comment, marked as answer]
You have to modify the templates/partials to do that.
Or possibly write a Lua filter that add contents.

See:
- https://quarto.org/docs/extensions/
- https://quarto.org/docs/journals/templates.html

Note that the author schema has the "roles" attribute: https://quarto.org/docs/journals/authors.html#author-schema

**Reply by k-soomro** [reply]
Thanks. I was able to modify the template to do it!
